python/ray/_private/runtime_env/packaging.py

```diff
@@ -883,3 +901,34 @@ def delete_package(pkg_uri: str, base_directory: str) -> Tuple[bool, int]:
             deleted = True
 
     return deleted
+
+
+async def install_wheel_package(
```

Comment:
why do we move to here?

Reply:
Initially I thought `working_dir` will support remote wheel as well but it isn't by our definition of working_dir.
But this function is generally used to install wheel package, so might as well move it outside from `py_modules` for general usage later.